src/DonationForms/Repositories/DonationFormDataRepository.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+namespace Give\DonationForms\Repositories;
+
+
+use Give\DonationForms\DonationFormDataQuery;
+use Give\DonationForms\Adapter\Form;
+use Give\DonationForms\ValueObjects\GoalType;
+
+class DonationFormDataRepository
```

Comment:
Let's add a Unit test for this, you can duplicate the one I started for campaigns and just swap it out for forms

https://github.com/impress-org/givewp/blob/ddf306c22d5def2f24dc11b7d2fd624cf7bd1498/tests/Unit/Campaigns/Repositories/CampaignsDataRepositoryTest.php#L19

Reply:
Resolved: https://github.com/impress-org/givewp/pull/7900/commits/a78d5c95d17e94202fb703d0f826eb03e9c8123d